client/src/components/NavBarLogin.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import PropTypes from "prop-types";
+
+const NavBarLogin = props => {
+  const { account, classes, setLoggedOutAccount } = props;
+
+  const loginLink = (
+    <li className={classes.userLogin}>
+      <Link className={`${classes.link} ${classes.lastItem}`} to="/login">
+        Login
+      </Link>
+    </li>
+  );
+
+  const userGreeting = (
+    <li className={classes.userLogin}>
+      <h4>Hello, {`${account.firstName} ${account.lastName} `}</h4>
```

Comment:
Test case complains that account may be null in this assignment.

Reply:
```suggestion
      <h4>Hello, {`${account ? account.firstName : ""} ${account ? account.lastName : ""} `}</h4>
```
Just a possible change that works but is kind of long